sci-geosciences/josm-bin/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>sci-geosciences@gentoo.org</email>
```

Comment:
Did you ask them to be added here? Perhaps you could maintain this yourself.

Reply:
good point, switched to "Proxy Maintainers"

Reply:
i was already in there